### DISCUSSION TITLE
How to copy data from one branch to another. dev > main

### DISCUSSION THREAD

**Discussion by dominic-vie** [planetscale/discussion#594] [Q&A] [answered]
I just imported a large dataset into the dev branch of about 100k records. I want to copy the data from dev to prod after I merge the schemas. How do I do that. 

I know merging the schemas has nothing to do with the data so I am trying to avoid having to reimport the dataset in production.

**Comment by orware** [comment, marked as answer]
Hello @dominic-vie,

I believe you received a reply already from your question in Discord, but for a more detailed example something along the lines of what is described in this discussion should be similar to what you would need to do:
https://github.com/planetscale/discussion/discussions/476#discussioncomment-6019692

The one extra tip that might be helpful to note is that for the `pscale database restore-dump` step it might be required to use the optional `--overwrite-tables` flag when overwriting the existing tables/data within your `prod` branch:
https://planetscale.com/docs/reference/database

I hope this is helpful!

**Comment by dominic-vie** [comment]
Thanks for your help and patience
